package.json

```diff
@@ -11,12 +11,12 @@
   "typings": "./dist/lib/js/index.d.ts",
   "license": "UNLICENSED",
   "engines": {
-    "yarn": "^1.22.0",
+    "yarn": "^1.22.1",
     "node": "^16.13.0 || ^18.17.0 || ^20.9.0"
   },
   "scripts": {
-    "build": "vue-cli-service build --target lib --formats umd --name design-system lib/js/index.ts",
-    "dev": "start-storybook -p 6006",
+    "build": "vite build",
```

Comment:
Previously we were building only umd, now we're building two formats:
```
dist/RadioButton-BSIC6h6x.mjs    1.76 kB │ gzip:   0.83 kB │ map:     3.92 kB
dist/pl-B2O3Uvpn.mjs             2.13 kB │ gzip:   0.96 kB │ map:     3.26 kB
dist/index-Oq81NaeJ.mjs         64.85 kB │ gzip:  16.54 kB │ map:   180.53 kB
dist/design-system.mjs         237.31 kB │ gzip:  25.77 kB │ map:     0.10 kB
dist/index-C_3Wau9G.mjs        771.47 kB │ gzip: 157.69 kB │ map: 4,713.17 kB
dist/design-system.umd.js  894.01 kB │ gzip: 179.64 kB │ map: 4,821.77 kB
```

How about building only one of them, whichever works best?